src/core/components/layout-utils.jsx

```diff
@@ -124,8 +125,7 @@ export class Button extends React.Component {
 
 }
 
-
-export const TextArea = (props) => <textarea {...props} />
+export const TextArea = (props) => <DebounceInput element="textarea" debounceTimeout={350} forceNotifyByEnter={false} {...props} />
```

Comment:
We have two options here - Introduce a new `DebounceTextArea` along with already existing `TextArea` or as a second option use `DebounceInput` in ad-hoc basis.

IMHO first option is preferable, as we can reuse code that way.

```js
export const DebounceTextArea = (props) => <DebounceInput element={TextArea} debounceTimeout={350} forceNotifyByEnter={false} {...props} />
```

Reply:
Additionally thank you for aligning with existing `350` ms debounce timeout which is consistent with the rest of the codebase. 

`forceNotifyByEnter={false}` seems idiomatic for debounced textareas as they allow editing multiline text.

Reply:
The default value for `forceNotifyByEnter` is `true` which I find a bit strange. I decided to set it to `false` to prevent triggering the "bounce" when adding a new line with the 'enter key' for performance reason. 

My spec file is 1.7MB (600+ requests with big models) and without this option the UI is laggy when adding new lines.